apps/sms/test/unit/thread_ui_test.js

```diff
@@ -4133,7 +4152,7 @@ suite('thread_ui.js >', function() {
             MockSettings.supportEmailRecipient = true;
 
             this.sinon.spy(ActivityPicker, 'email');
-            this.sinon.spy(ActivityPicker, 'sendMessage');
+            this.sinon.spy(ThreadUI, 'navigateToComposer');
```

Comment:
I'm asserting this method instead of ```Navigation.toPanel``` below just to be sure that we support "discard" confirm dialog for those cases and not just navigation to Composer